### PR TITLE
Change GAX to GAPIC, which is more appropriate

### PR DESCRIPTION
**Pull request by bjwatson**

### CODE REVIEW

**Comment by garrettjonesgoogle** [comment]
LGTM

**Comment by bjwatson** [comment]
@geigerj Packages are now available. I'll need to check with you on how we do the docs lately.

**Comment by bjwatson** [comment]
@geigerj Since we're too busy to update docs right now, I reverted that portion of this PR so we can get it merged. PTAL

File: PACKAGES.md [hunk 1]
```diff
@@ -25,25 +25,27 @@ found. Currently, package delivery is only being done for Python.
...
+- [GAPIC](https://pypi.python.org/pypi/gapic-google-pubsub-v1)
 - [gRPC](https://pypi.python.org/pypi/grpc-google-pubsub-v1)
-- [Documentation](http://pythonhosted.org/gax-google-pubsub-v1/)
+- [Documentation](http://pythonhosted.org/gax-google-pubsub-v1/):
+  For old `gax` namespace... will be updated soon for `gapic` namespace.
 
 ### [Cloud Speech](https://cloud.google.com/speech/)
 - *NOTE: v1 is the wrong version, and these will soon be replaced by v1beta1*
-- [GAX](https://pypi.python.org/pypi/gax-google-cloud-speech-v1)
+- [GAPIC](https://pypi.python.org/pypi/gapic-google-cloud-speech-v1)
```
Comment (geigerj) [inline PACKAGES.md]: Per offline discussion, this link will go live shortly.

**Pull request merged**